src/pudl/helpers.py

```diff
@@ -1009,19 +1009,19 @@ def remove_leading_zeros_from_numeric_strings(
     return df
 
 
-def fix_eia_na(df: pd.DataFrame) -> pd.DataFrame:
+def fix_na(df: pd.DataFrame) -> pd.DataFrame:
     """Replace common ill-posed EIA NA spreadsheet values with np.nan.
```

Comment:
If we're changing this function name, maybe something more descriptive like `standardize_na_values()` would be better than "fix" since I could see fixing NA values meaning filling them in with non-null values.

Reply:
Updated